Migration-From-v1-To-v2.md

```diff
@@ -0,0 +1,151 @@
+# Migration from v1 to v2
+
+## Before You Start
+
+- Stop your Uptime Kuma and:
+   - Backup your `data` directory.
+   - Make sure you have a backup of your `data` directory again.
+   - Make sure you have a backup of your `data` directory again again.
+- The migration process could take some time to complete, depending on the size of your database.
+   - You should be able to view the logs of the migration process in the console.
+   - Do not interrupt the migration process.
+   - FYI: My Uptime Kuma had 20 monitors and 90 days of data, and it took around 7 minutes to migrate.
+- Beta version is not stable and may contain bugs, especially for the first beta release.
+
+
+## Breaking Changes
+
+- The `:duration` of these endpoints accepts `24`, `24h`, `30d`, `1y` only
+  - <kuma-url>/api/badge/:monitorID/ping/:duration
+  - <kuma-url>/api/badge/:monitorID/uptime/:duration
+- Dropped support for legacy browsers
+- The deprecated backup feature is completely removed now
```

Comment:
I went through the PRs and have added a few other breaking changes

```suggestion
- The `:duration` of these endpoints accepts `24`, `24h`, `30d`, `1y` only
  - `<kuma-url>/api/badge/:monitorID/ping/:duration`
  - `<kuma-url>/api/badge/:monitorID/uptime/:duration`
- Dropped support for legacy browsers
- The deprecated [backup](https://github.com/louislam/uptime-kuma/pull/3892) and [CachableLookup](https://github.com/louislam/uptime-kuma/issues/3762) features are completely removed now
- Default retires for **NEW**ly created monitors are now `0` instead of `1` to prevent users being confused
- `Email (SMTP)` switched templating for subjects/bodys from a custom-regex to [LiquidJS](https://liquidjs.com/).
  LiquidJS **does discriminate between case-ness** of variables and **ignores all non-matching variables**.
  These are the supported variables: `name`, `msg`, `status`, `heartbeatJSON`, `monitorJSON`, `hostnameOrUrl`
```